tests/conftest.py

```diff
@@ -123,7 +123,9 @@ def _get_pseudo_family(
         label='family',
         cls=PseudoPotentialFamily,
         pseudo_type=PseudoPotentialData,
-        elements=None
+        elements=None,
+        cutoffs=None,
```

Comment:
Maybe still add `cutoffs` and `default_stringency` to the docstring? These input arguments also don't seem to be used anywhere at the moment (unless I missed it). 😅